src/utils.ts

```diff
@@ -258,24 +259,20 @@ export async function askToConfigureEnvironment(
           message, DialogResponses.yes, DialogResponses.no);
 
   if (result === DialogResponses.yes) {
-    telemetryContext.properties.errorMessage =
-        `${operation} operation failed and user configures project`;
-
-    let res: boolean;
-    res =
-        await ProjectEnvironmentConfiger.configureProjectEnvironmentAsPlatform(
-            context, channel, telemetryContext, platform, deviceRootPath,
-            scaffoldType);
-    if (res) {
-      const message =
-          `Configuration of project environmnet done. You can run the ${
-              operation.toLocaleLowerCase()} operation now.`;
-      channelShowAndAppendLine(channel, message);
-      vscode.window.showInformationMessage(message);
-    }
+    telemetryContext.properties.errorMessage = `${
+        operation} operation failed and user configures project environment.`;
+
+    await ProjectEnvironmentConfiger.configureProjectEnvironmentAsPlatform(
+        context, channel, telemetryContext, platform, deviceRootPath,
+        scaffoldType);
+    const message =
+        `Configuration of project environmnet done. You can run the ${
+            operation.toLocaleLowerCase()} operation now.`;
+    channelShowAndAppendLine(channel, message);
+    vscode.window.showInformationMessage(message);
   } else {
     const message = `${
-        operation} operation failed since project development environment configuration cancelled.`;
+        operation} operation failed and user cancels to configure project environment.`;
```

Comment:
This may be 'canceled' since 'failed' is the past tense

Reply:
These are two seperate parts of event. The first operation has failed. And the failed operation asks user to do some choice. 
Thus, we say "the operation failed" and "user choose to do something".
This tense should be okay.